performant-software/dm-2
# Toggle text highlights on and off with "eye" icon

### What this PR does
- #293
  - Allows user to toggle text highlights in addition to image highlights
- Closes #304, allowing user to toggle text highlights on each instance separately
- Corrects typos in lock validation and delete document controller functions
- Bumps `canvas` from 2.6.0 to 2.7.0 (non-breaking) to fix `node-gyp-pre` error

### Additional notes

- Like with images, highlights toggled off on text documents will not reappear when one "Checks Out" the document. It should be determined if this is intended behavior.
  
### Status


### Steps to test

- Visit https://dm-2-staging.herokuapp.com/
- Log in
- Open or create a project
- Open or create a text document with some highlights
- If the document is checked out, click the "Check In" button
- Click the "eye" button in the top-right corner of the document to hide the highlights
- Click the "eye" button again to show highlights
